docs/reference/content/getting-started/quick-tour.md

```diff
@@ -0,0 +1,381 @@
++++
+date = "2015-03-17T15:36:56Z"
+title = "Quick Tour"
+[menu.main]
+  parent = "Getting Started"
+  identifier = "Quick Tour"
+  weight = 30
+  pre = "<i class='fa'></i>"
++++
+
+# Quick Tour
+
+The following code snippets come from the `QuickTour.scala` example code that can be found with the 
+[driver source]({{< srcref "examples/src/test/scala/tour/QuickTour.scala">}}).
+
+{{% note %}}
+See the [installation guide]({{< relref "getting-started/installation-guide.md" >}})
+for instructions on how to install the MongoDB Scala Driver.
+
+This guide uses the `Helper` implicits as covered in the [Quick Tour Primer]({{< relref "getting-started/quick-tour-primer.md" >}}).
```

Comment:
The primer makes more sense coming to it from here instead of starting with it before seeing this page first.